raiden/tests/integration/api/test_restapi.py

```diff
@@ -359,9 +360,19 @@ def test_api_open_and_deposit_channel(api_server_test_instance, token_addresses,
         }
     )
     assert check_dict_nested_attrs(json_response, expected_response)
-
     token_network_address = json_response["token_network_address"]
 
+    # Now let's try to open the same channel again and see that a proper error is returned
+    request = grequests.put(
+        api_url_for(api_server_test_instance, "channelsresource"), json=channel_data_obj
+    )
+    response = request.send().response
+    assert_proper_response(response, HTTPStatus.CONFLICT)
+    json_response = get_json_response(response)
+    assert first_partner_address in json_response["errors"]
+    assert token_address_hex in json_response["errors"]
+    assert "b\\" not in json_response["errors"]
```

Comment:
At least from reading the bugreport, this should be
```
    assert "b'\\" not in json_response["errors"]
```
I would even suggest to go for 
```suggestion
    assert "b'" not in json_response["errors"]
```
so we'd catch any byte string.

Reply:
This test caught the bug when I ran it.